### PR TITLE
add tftp_managed parameter

### PR DESCRIPTION
**Pull request by mmoll**
with this I can use -installer also with libvirt (which brings its own tftp server, but I still want the syslinux files to get copied)

### CODE REVIEW

**Comment by brandonweeks** [comment]
LGTM.

File: manifests/config.pp [hunk 1]
```diff
@@ -8,7 +8,11 @@
     Class['puppet::server::config'] ~> Class['foreman_proxy::service']
   }
 
-  if $foreman_proxy::tftp     { include ::foreman_proxy::tftp }
+  if $foreman_proxy::tftp {
+    class { '::foreman_proxy::tftp':
+      tftp_managed => $foreman_proxy::tftp_managed,
```
Comment (domcleal) [inline manifests/config.pp]: I don't think this needs to be set/duplicated here, the default's already set correctly.

Comment (mmoll) [inline manifests/config.pp]: indeed, was a leftover from drafting the change

**Comment by domcleal** [comment]
:+1:

**Comment by brandonweeks** [comment]
Merged, thanks @mmoll. 😃

**Pull request merged**